components/tooltip/src/tooltip.tsx

```diff
@@ -19,7 +19,7 @@ const TooltipContent = forwardRef<ElementRef<typeof Content>, ComponentPropsWith
 			ref={ref}
 			sideOffset={sideOffset}
 			className={cx(
-				"z-50 overflow-hidden rounded-md border bg-popover px-3 py-1.5 text-sm text-popover-foreground shadow-md animate-in fade-in-0 zoom-in-95 data-side-bottom:slide-in-from-top-2 data-side-left:slide-in-from-right-2 data-side-right:slide-in-from-left-2 data-side-top:slide-in-from-bottom-2 data-state-closed:animate-out data-state-closed:fade-out-0 data-state-closed:zoom-out-95",
+				"z-50 overflow-hidden rounded-md border bg-popover px-3 py-1.5 text-sm text-popover-foreground shadow animate-in fade-in-0 zoom-in-95 data-side-bottom:slide-in-from-top-2 data-side-left:slide-in-from-right-2 data-side-right:slide-in-from-left-2 data-side-top:slide-in-from-bottom-2 data-state-closed:animate-out data-state-closed:fade-out-0 data-state-closed:zoom-out-95",
```

Comment:
I'm ok with the smaller shadow here

Reply:
Weird! I don't think I intended to change that. But I also think it's fine.